thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/context/AnomalyResult.java

```diff
@@ -0,0 +1,87 @@
+package com.linkedin.thirdeye.anomalydetection.context;
```

Comment:
Considering grouping, should we add a list of containing anomalies or an enum showing the level of the anomaly?